server/ControlPlane/Compute/Kubernetes/LoggerExtensions.cs

```diff
@@ -32,8 +32,8 @@ public static partial class LoggerExtensions
     [LoggerMessage(LogLevel.Information, "Finalizing run {runId}.")]
     public static partial void FinalizingRun(this ILogger logger, long runId);
 
-    [LoggerMessage(LogLevel.Information, "Finalized run {runId}.")]
-    public static partial void FinalizedRun(this ILogger logger, long runId);
+    [LoggerMessage(LogLevel.Information, "Finalized run {runId}. Status: {status}, CreateTime: {createTime}, StartTime: {startTime}, FinishTime: {finishTime}")]
```

Comment:
For consistency with the code and database names, I suggest `createdAt`, `startedAt`, and `finishedAt`.